src/windows/wsladiag/wsladiag.cpp

```diff
@@ -44,15 +47,20 @@ int wsladiag_main(std::wstring_view commandLine)
     bool help = false;
     bool list = false;
 
+    std::wstring debugShell;
+
     parser.AddArgument(list, L"--list");
     parser.AddArgument(help, L"--help", L'h'); //  short option is a single wide char
+    parser.AddArgument(debugShell, L"--debug-shell");
+
     parser.Parse();
 
     auto printUsage = []() {
         wslutil::PrintMessage(
             L"wsladiag - WSLA diagnostics tool\n"
             L"Usage:\n"
             L"  wsladiag --list    List WSLA sessions\n"
+            L"  wsladiag --debug-shell <SessionName>  Open a debug shell in an existing WSLA session\n"
```

Comment:
`debug-shell` is a term that we used in the temporary wsl.exe tool, for this I'd recommend just using `shell` to make things easier